gcpkms/keysource.go

```diff
@@ -238,8 +238,10 @@ func (key *MasterKey) newKMSClient() (*kms.KeyManagementClient, error) {
 // JSON format. It returns an error if the file cannot be read, and may return
 // a nil byte slice if no value is set.
 func getGoogleCredentials() ([]byte, error) {
-	defaultCredentials := os.Getenv(SopsGoogleCredentialsEnv)
-	if _, err := os.Stat(defaultCredentials); err == nil {
+	defaultCredentials, wasSet := os.LookupEnv(SopsGoogleCredentialsEnv)
+	if !wasSet {
+		return nil, nil
+	} else if _, err := os.Stat(defaultCredentials); err == nil {
 		return os.ReadFile(defaultCredentials)
 	}
 	return []byte(defaultCredentials), nil
```

Comment:
```suggestion
	if defaultCredentials, ok := os.LookupEnv(SopsGoogleCredentialsEnv); ok {
		if _, err := os.Stat(defaultCredentials); err == nil {
			return os.ReadFile(defaultCredentials)
		}
		return []byte(defaultCredentials), nil
	}
	return nil, nil
```

Seems more Go idiomatic.